src/source/mod.rs

```diff
@@ -242,6 +243,42 @@ pub trait Source: Iterator<Item = Sample> {
         amplify::amplify(self, value)
     }
 
+    /// Amplifies the sound logarithmically by the given value.
+    #[inline]
+    fn amplify_decibel(self, value: f32) -> Amplify<Self>
+    where
+        Self: Sized,
+    {
+        amplify::amplify(self, to_linear(value))
+    }
+
+    /// Normalized amplification in `[0.0, 1.0]` range. This method better matches the perceived
```

Comment:
For proper attribution, I suggest adding a link to the source of the algorithm: https://www.dr-lex.be/info-stuff/volumecontrols.html